src/models/encrypted-model.ts

```diff
@@ -18,15 +19,25 @@ export abstract class EncryptedModel extends Model {
 		if (typeof value !== "string") {
 			throw new Error(`Cannot decrypt '${field}', it is not a string.`);
 		}
-		return await EncryptionClient.decrypt(value, await this.getEncryptContext(field));
+		try {
+			return await EncryptionClient.decrypt(value, await this.getEncryptContext(field));
+		} catch (e) {
+			getLogger("cryptor").error(`Fail to decrypt field ${field}`, { error: e });
+			throw e;
+		}
 	}
 
 	protected async encrypt(field: (keyof StringValues<this>)): Promise<string> {
 		const value = this[field];
 		if (typeof value !== "string") {
 			throw new Error(`Cannot encrypt '${field}', it is not a string.`);
 		}
-		return await EncryptionClient.encrypt(this.getEncryptionSecretKey(), value, await this.getEncryptContext(field));
+		try {
+			return await EncryptionClient.encrypt(this.getEncryptionSecretKey(), value, await this.getEncryptContext(field));
+		} catch (e) {
+			getLogger("cryptor").error(`Fail to encrypt field ${field}`, { error: e });
```

Comment:
set the logger as a constant, no need to keep recreating it...

Reply:
fixed.